.github/workflows/build_application_images.yml

```diff
@@ -75,18 +76,19 @@ jobs:
       - name: Login to Amazon ECR
         id: login-ecr
         uses: aws-actions/amazon-ecr-login@v1
+      - name: Setup Docker Buildx
+        uses: docker/setup-buildx-action@v3
       - name: Build, tag, and push image to Amazon ECR
```

Comment:
Is this still pushing to ECR as is? The old action called `docker image push` and this one's just using `push: true`, but I don't see where we're passing the ECR registry/repository

Reply:
push: true 
with
tags: ${{ steps.login-ecr.outputs.registry }}/easi-db-migrate:${{ env.GIT_HASH }}

@ClayBenson94

Reply:
![image](https://github.com/CMSgov/easi-app/assets/15203744/018b6121-0ecc-40d0-93e0-103e8e095518)
Looks like it, though 😄 all good!

Reply:
Yeah I stole from this code that I replaced with push: true and the tag
`        env:
          ECR_REGISTRY: ${{ steps.login-ecr.outputs.registry }}
          ECR_REPOSITORY: easi-db-migrate`